Lib/http/server.py

```diff
@@ -100,6 +100,7 @@
 import socketserver
 import sys
 import time
+import datetime
```

Comment:
Done in commit 708ff03. The order was not supported for argparse and copy...
I also changed to version to 0.7.